hedera-node/cli-clients/src/main/java/com/hedera/services/cli/ServiceSignCommand.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2016-2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.cli;
+
+import com.swirlds.cli.PlatformCli;
+import com.swirlds.cli.utility.AbstractCommand;
+import com.swirlds.cli.utility.SubcommandOf;
+import picocli.CommandLine;
+
+/**
+ * A collection of operations on signing all necessary stream files generated by services.
+ */
+@CommandLine.Command(
+        name = "service-sign",
```

Comment:
The command structure chosen here works, but it's inverted from the way existing platform commands are laid out:

Existing platform command structure:
`pcli <object> <operation> <...>` -> `pcli event-stream sign <...>`

Command structure in this PR:
`pcli <operation> <object> <...>` -> `pcli service-sign balance <...>`

The following changes would bring your structure in line with existing commands:
- Instead of `ServiceSignCommand`, you would have an `AccountBalanceCommand` class, which extends `AbstractCommand`, is tagged `@SubcommandOf(PlatformCli.class)`, with `name = account-balance`
- The `BalanceFileSignCommand` would be tagged `@SubcommandOf(AccountBalanceCommand)`, extend `SignCommand`, with `name = sign`

Therefore, the command to use the tool would be
```
pcli account-balance sign <...> 
```

Reply:
Services have two types of stream files to sign at this time, pcli serive-sign balance and record-stream. Maybe more later. So I setup service-sign at the top level.

Reply:
The idea is that all signing methods share the same command, `sign`. This is fully compatible with having multiple types of stream files to sign.

The current structure is not consistent with the intended usage

Reply:
I updated the code. Now you can run it as : pcli account-balance sign XXX